src/Integration/Connection/ConnectionWorkflow.cs

```diff
@@ -26,17 +25,20 @@ namespace SonarLint.VisualStudio.Integration.Connection
     /// </summary>
     internal class ConnectionWorkflow
```

Comment:
Another goal was to break the dependency of the workflows on the controllers whilst still consuming the needed services which are now exposed by the host.